src/gei/Commands/ReclaimMannequinCommand.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Threading.Tasks;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter.Commands
+{
+    public class ReclaimMannequinCommand : Command
+    {
+        private readonly OctoLogger _log;
+        private readonly ITargetGithubApiFactory _targetGithubApiFactory;
+
+
+        private readonly EnvironmentVariableProvider _environmentVariableProvider;
+
+        public ReclaimMannequinCommand(OctoLogger log, ITargetGithubApiFactory targetGithubApiFactory, EnvironmentVariableProvider environmentVariableProvider) : base("reclaim-mannequin")
+        {
+            _log = log;
+            _targetGithubApiFactory = targetGithubApiFactory;
+            _environmentVariableProvider = environmentVariableProvider;
+
+            Description = "Reclaims a mannequin user. An invite will be sent and the user will have to accept for the remapping to occur.";
+
+            var githubTargetOrgOption = new Option<string>("--github-target-org")
+            {
+                IsRequired = true,
+                Description = "Uses GH_PAT env variable."
+            };
+
+            var mannequinUsernameOption = new Option<string>("--mannequin-user")
+            {
+                IsRequired = true,
+                Description = "The login of the mannequin to be remapped."
+            };
+            var targetUsernameOption = new Option<string>("--target-user")
+            {
+                IsRequired = true,
+                Description = "The login of the target user to be mapped."
+            };
+
+            var forceOption = new Option("--force")
+            {
+                IsRequired = false,
+                Description = "Map the user even if it was previously mapped"
+            };
+
+
+            var verbose = new Option("--verbose")
+            {
+                IsRequired = false
+            };
+
+            AddOption(githubTargetOrgOption);
+            AddOption(mannequinUsernameOption);
+            AddOption(targetUsernameOption);
+            AddOption(forceOption);
+            AddOption(verbose);
+
+            Handler = CommandHandler.Create<string, string, string, bool, bool>(Invoke);
+        }
+
+        public async Task<bool> Invoke(
```

Comment:
what does the bool return value mean? we don't do that in any other tasks

Reply:
There was no prior art, so I decided to use a boolean, but come to think of it an exception is probably better.

What exception should we return on failure?

Reply:
@dylan-smith refactored the code in [41c90f7](https://github.com/github/gh-gei/pull/271/commits/41c90f7cdf07e94f08efa1c5d1ce0d9bc3378a9b). It no longers returns a boolean to make the command consistent

The API call is now also strongly type so we can show the user the error message sent from the server if an error occurs